src/Workspaces/Core/MSBuild/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj

```diff
@@ -16,44 +22,103 @@
     </PackageDescription>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build" Version="$(RefOnlyMicrosoftBuildVersion)" ExcludeAssets="Runtime" PrivateAssets="All" />
-    <PackageReference Include="Microsoft.Build.Framework" Version="$(RefOnlyMicrosoftBuildFrameworkVersion)" ExcludeAssets="Runtime" />
-    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(RefOnlyMicrosoftBuildTasksCoreVersion)" ExcludeAssets="Runtime" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Build" Version="$(RefOnlyMicrosoftBuildVersion)" />
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(RefOnlyMicrosoftBuildFrameworkVersion)" />
+    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(RefOnlyMicrosoftBuildTasksCoreVersion)" />
```

Comment:
Do we want these to become transitive dependencies? Wouldn't they be loaded by the BuildHost from the MSBuild directory?

Reply:
These are now fine to be transitive, since ILogger is still on the surface area (and also we might still want to use MSBuild inproc for things like SLN reading...)

Reply:
Ah, ok. ILogger. Was just thinking about packaging and deploying dotnet-format. I guess we currently have these as top-level dependencies and exclude them. So likely no change necessary on our end there.

Reply:
@JoeRobich: no change should be necessary. If you are an app that was not deploying them and using MSBuildLocator, then you're still fine (it was just unnecessary to do that!); if you were deploying them it's still fine now too.